### PR TITLE
updated aframe version to 0.7.0

### PR DESCRIPTION
**Pull request by m-ke**
In A-frame 0.7.0 getComputedAttribute is deprecated and causes a lot of warnings

### CODE REVIEW

**Comment by AVGP** [comment]
This was very helpful to get rid of all the warning logs. @atomicguy could you merge this please?

**Comment by atomicguy** [comment]
sorry for the slow response. I'll take a look and merge asap

File: dist/aframe-fence-component.js [hunk 1]
```diff
@@ -42,7 +42,7 @@
 /************************************************************************/
 /******/ ([
 /* 0 */
-/***/ function(module, exports) {
+/***/ (function(module, exports) {
```
Comment (atomicguy) [inline dist/aframe-fence-component.js:45]: just curious, why do you wrap the function in ()?

Comment (m-ke) [reply dist/aframe-fence-component.js:45]: to be honest with you it was most probably my IDE deciding it was a good idea and me making the commit without realizing :) I can remove it.

Comment (atomicguy) [reply dist/aframe-fence-component.js:45]: no worries. looks good to me

**Pull request merged**